setup.sh

```diff
@@ -157,21 +157,19 @@ export ASB_NODEPOOLS_SUBNET_ID=$(az deployment group show -g $ASB_RG_SPOKE -n sp
 az deployment group create -g $ASB_RG_HUB -f networking/hub-regionA.json -p location=${ASB_LOCATION} nodepoolSubnetResourceIds="['${ASB_NODEPOOLS_SUBNET_ID}']"
 export ASB_SPOKE_VNET_ID=$(az deployment group show -g $ASB_RG_SPOKE -n spoke-BU0001A0008 --query properties.outputs.clusterVnetResourceId.value -o tsv)
 
-# create ARM template
-rm -f cluster-${ASB_TEAM_NAME}.json
-# file contains '$schema'
-cat templates/cluster-stamp.json | envsubst '$ASB_TEAM_NAME,$ASB_DNS_ZONE,$ASB_TLD' > cluster-${ASB_TEAM_NAME}.json
-
 # grant executer permission to the key vault
 az keyvault set-policy --certificate-permissions list get --object-id $(az ad signed-in-user show --query objectId -o tsv) -n $ASB_KV_NAME -g TLD
 az keyvault set-policy --secret-permissions list get --object-id $(az ad signed-in-user show --query objectId -o tsv) -n $ASB_KV_NAME -g TLD
 
 # create AKS
 az deployment group create -g $ASB_RG_CORE \
-  -f  cluster-${ASB_TEAM_NAME}.json \
```

Comment:
Should we move the file up a level since it's not a template any longer?

This would make it easier to PR upstream.

Reply:
done!